AsyncGenerator.yml

```diff
@@ -1,5 +1,6 @@
 ﻿projects:
 - filePath: CoreMemoryCache\NHibernate.Caches.CoreMemoryCache\NHibernate.Caches.CoreMemoryCache.csproj
+  targetFramework: netstandard2.0
```

Comment:
I guess it is required now that it does multi-target, but I cannot test it, AsyncGenerator being broken once VS 15.6.3 is installed. `nant generate-async` encounters the same error than the async generation for NHibernate.